lib/state.js

```diff
@@ -437,9 +454,8 @@ StateResolver.prototype.resolveMirrorFast_ = function(mirror) {
 
 StateResolver.prototype.getMirrorProperties_ = function(mirror) {
   var numProperties = this.config_.capture.maxProperties;
-  var namedProperties = mirror.properties(1, numProperties);
-  var indexedProperties = mirror.properties(2, numProperties);
-  return namedProperties.concat(indexedProperties);
+  var properties = mirror.properties();
```

Comment:
Does this work with all node versions? I am curious why the original code is the way it is.